api-design-overview.md

```diff
@@ -0,0 +1,30 @@
+# SuperFormula Back End design test
+
+## Environment
+
+- AWS Cloud Native
+- API Gateway per-route entry points
+- CRUD routes trigger Lambdas
+- DynamoDB storage
+- DLQ for catching errors
+- Infrastructure as configuration/code
+
+## Questions
+
+1. Do we need to secure the API? (API key, lambda authorizer, something else?)
```

Comment:
Great question. Please secure the API with an authorizer lambda, if time allows.